cns/restserver/ipam.go

```diff
@@ -287,6 +297,7 @@
 	ipconfig.OrchestratorContext = nil
 	service.PodIPConfigState[ipconfig.ID] = ipconfig
 	delete(service.PodIPIDByOrchestratorContext, podInfo.GetOrchestratorContextKey())
+	logger.Printf("Deleted outdated pod info %s from PodIPIDByOrchestratorContext since IP will be released", podInfo.GetOrchestratorContextKey())
```

Comment:
We should also log what was the ip associated to this Pod

Something like "Deleting PodContext %s . Marking allocated ip %s to available"

Try to add logs which you think will help during debugging

Reply:
IP info is logged in its caller. Though I can add here too for easier debug.